tests/api/search/search_entities.test.js

```diff
@@ -61,6 +64,13 @@ describe('search:entities', () => {
         })
       })
 
+      it('should not return results that do not contain the exact searched words', async () => {
```

Comment:
this test seems to be in contradiction with the comment line 53 

`// Ex: an exact search for "Charles Ben" will find "Charles Bent" as one of its generated tokens is "Ben"` 

?

Reply:
solved by b73e06d11